scripts/run-serialization-roundtrip-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+export PYTHONPATH="$BUILD_DIR/lib/python"
```

Comment:
You're assuming `$BUILD_DIR` is set. Basically you're assuming orchestra.
I would assume instead that `$PWD` is the one set by `add_test`.